js/utils.js

```diff
@@ -17,6 +17,55 @@ Fliplet.Registry.set('dynamicListUtils', (function () {
   };
   var computedFieldClashes = [];
 
+  function setLineClamps () {
+    setClampHeight()
+      .then(function (elements) {
+        Superclamp.register(elements);
+        return Promise.resolve(elements);
+      })
+      .then(function (elements) {
+        removeClampHeight(elements);
+      })
+      .catch(function (selector) {
+        console.error('Fail to clamp line in the selector ' + selector);
+      });
+  }
+
+  function setClampHeight (selector) {
+    selector = selector || '[data-line-clamp]';
+
+    return new Promise(function (resolve, reject) {
+      var elements = document.querySelectorAll(selector);
```

Comment:
I suggest using jQuery as it supports a wider range of selectors.

```suggestion
      var $elements = $(selector);
```